biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java

```diff
@@ -76,6 +78,46 @@
  */
 public class JPMSModuleInfoPlugin implements VerifierPlugin {
 
+	enum Access {
+		CLOSED(
```

Comment:
I hope to god the next Eclipse version fixes this idiotic code formatting bug...

Reply:
:)